Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -10,6 +10,25 @@
 
 DECLARE_LOG_CATEGORY_EXTERN(LogBenchmarkGymGameModeBase, Log, All);
 
+class FPrintTimer
+{
+public:
+
+	FPrintTimer() = default;
```

Comment:
Should we remove this and enforce initilization with a time?

Reply:
Ahh yes, forgot it's not an unreal class.